iroh/src/magicsock.rs

```diff
@@ -1826,6 +1801,23 @@ impl Handle {
             }
             .instrument(info_span!("relay-actor")),
         );
+        actor_tasks.spawn({
+            let msock = msock.clone();
+            async move {
+                while let Some(message) = relay_disco_recv_rx.recv().await {
+                    msock.handle_disco_message(
+                        message.source,
+                        &message.sealed_box,
+                        DiscoMessageSource::Relay {
+                            url: message.relay_url,
+                            key: message.relay_remote_node_id,
+                        },
+                    );
+                }
+                debug!("relay-disco-recv actor closed");
```

Comment:
nit: "actor closed" is sufficient since you already have an info span with the actor name.  But I don't mind if you do this either.